.github/workflows/test_and_deploy_booking_app.yaml

```diff
@@ -0,0 +1,41 @@
+name: "Test and build Booking app"
```

Comment:
Same as above. Preferable the 

- build pipeline 
    - should produce the artefacts
    - should be triggered by a tag creation
- deploy pipeline should 
    -  deploy the artefacts
    -  should be triggered by the completion of the build pipeline

Reply:
Do we want to deploy the booking app to the store/testflight? Even for android?